test/svg-sprite/transform/svgo.test.js

```diff
@@ -0,0 +1,212 @@
+'use strict';
+
+const svgo = require('svgo');
+const svgoTransform = require('../../../lib/svg-sprite/transform/svgo.js');
+
+const TEST_SVG = '<svg></svg>';
+
+jest.mock('svgo');
+
+describe('testing transforms.svgo', () => {
+    let noop;
+
+    beforeEach(() => {
+        noop = jest.fn();
```

Comment:
Isn't `noop` the same across the tests? If so, shouldn't we use the same instance/variable?

Reply:
not it isn't. before each test it replaces with new instance of jest.fn. Additionally, all the mocks are cleared and reset after each test.